streaming/streaming.go

```diff
@@ -290,3 +320,160 @@ func fileExists(fileName string) bool {
 	}
 	return !info.IsDir()
 }
+
+func extractTheClaimsOutOfTSDToken(response *http.Response) (string, jwt.MapClaims, error) {
```

Comment:
Too long name, maybe something like `extractClaims` would be better. If needed, more comments could be added to make it more clear.

Reply:
@a-ghanem 
Done

Reply:
in [e868ec3](https://github.com/elixir-oslo/lega-commander/pull/5/commits/e868ec38d674dc9b4c21b553f7a3efad3c866693)